pkg/tsdb/cloudwatch/test_utils.go

```diff
@@ -192,3 +201,10 @@ func (s fakeSessionCache) GetSession(c awsds.SessionConfig) (*session.Session, e
 		Config: &aws.Config{},
 	}, nil
 }
+
+func pointerString(s string) *string {
+	return &s
+}
+func pointerInt64(i int64) *int64 {
+	return &i
+}
```

Comment:
probably not needed for this PR but the method `aws.String` and `aws.Int64` can be used rather than redefining them.

Reply:
Thanks